Project/Source/Components/ComponentMeshRenderer.cpp

```diff
@@ -562,10 +603,10 @@ void ComponentMeshRenderer::Draw(const float4x4& modelMatrix) const {
 		int hasSpecularMap = specular ? 1 : 0;
 
 		// Specular-specific uniform settings
-		if (goBones.size()) {
-			program = App->programs->specularSkinning;
+		if (hasNormalMap) {
+			program = App->programs->specularNormal;
 		} else {
-			program = App->programs->specularNotSkinning;
+			program = App->programs->specularNormal;
```

Comment:
This should probably be specularNotNormal

Reply:
Totally right